framework/doc/content/source/meshdivisions/SubdomainsDivision.md

```diff
@@ -2,9 +2,15 @@
 
 !syntax description /MeshDivisions/SubdomainsDivision
 
+If not setting the [!param](/MeshDivisions/SubdomainsDivision/block) parameter:
 The divisions are numbered from 0 to the number of blocks in the mesh minus one.
 The ordering of the divisions is the same of the ordering of the IDs of the subdomains.
 
+If specifying a block restriction using the [!param](/MeshDivisions/SubdomainsDivision/block) parameter:
+The divisions are numbered from 0 to the number of blocks specified. An invalid id is returned
```

Comment:
I would expect the math here to be similar. So 0 to the number of blocks specified minus one is what I would expect